settings.py.template

```diff
@@ -24,7 +30,7 @@ PYLTI_CONFIG = {
 
 # Secret key used for Flask sessions, etc. Must stay named 'secret_key'.
 # Can be any randomized string, reccomend generateing one with os.urandom(24)
```

Comment:
Since you're editing this file already, could you please fix the typos here? (reccomend generateing)
Thanks